core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -103,6 +103,9 @@ object ApiVersion {
     KAFKA_2_7_IV0,
     // Bup Fetch protocol for Raft protocol (KIP-595)
     KAFKA_2_7_IV1,
+    // Enable redirection (KIP-590)
+    // TODO: remove this IBP in the 2.7 release if redirection work could not be done before the freeze
```

Comment:
Get rid of this TODO. We do not need to remove IBP internal versions.

Reply:
But in case we release AK 2.7, wouldn't this flag give user the confidence to upgrade to, which we don't want to happen?

Reply:
I'm not sure I follow. Do you not want redirection to be part of 2.7?